src/js/administration/types.ts

```diff
@@ -1,13 +1,24 @@
+/**
+ * Full model of an administrator role
+ */
 export type AdministratorRole = {
     id: string;
     name: AdministratorRoles;
     description: string;
 };
 
+/**
+ * All administrator roles
+ */
 export enum AdministratorRoles {
     FULL = "full",
     SETTINGS = "settings",
     SPACES = "spaces",
     USERS = "users",
-    BASE = "base"
+    BASE = "base",
 }
+
+export type permissionQueryResult = {
```

Comment:
I believe this is conventional.
```suggestion
export type PermissionQueryResult = {
```